src/Pathy/Printer.purs

```diff
@@ -25,7 +25,7 @@ import Data.String.NonEmpty.CodeUnits (cons, singleton) as NES
 import Partial.Unsafe (unsafePartial)
 import Pathy.Name (Name)
 import Pathy.Path (Path, foldPath, (</>))
-import Pathy.Phantom (class IsDirOrFile, class IsRelOrAbs, Dir, Rel, foldDirOrFile, foldRelOrAbs, kind DirOrFile, kind RelOrAbs)
+import Pathy.Phantom (class IsDirOrFile, class IsRelOrAbs, Dir, Rel, foldDirOrFile, foldRelOrAbs)
```

Comment:
Just now getting to this. 

Could you explain why the two kinds here were removed? Is this because you're also updating this lib for the v0.14.0-rc2 release?

Reply:
It's just because they aren't used in this code, and so this produced a compiler warning. Nothing to do with 0.14!

Reply:
I think kinds weren't considered in the import usage checking previously, which is why it became a warning now.

Reply:
Gotcha. Thanks for clarifying.